frontend/src/components/Layouts/HomePageComponent/components/Carousel/Carousel.module.scss

```diff
@@ -51,7 +51,7 @@
 }
 @include sm {
   .carousel {
-    margin: 0 16px;
+    margin: 0 16px 0 0;
 
```

Comment:
We had earlier margin-left and margin-right = 16px. Now we have only margin-right =16px. As I can see, both margin-right and margin-left should be now 0. Otherwise, the carousel is not in the center